pkg/storaged/client.js

```diff
@@ -434,15 +434,7 @@
                     return cockpit.resolve();
                 },
                 function () {
-                    return PK.call("/org/freedesktop/PackageKit", "org.freedesktop.DBus.Properties",
-                                   "Get", [ "org.freedesktop.PackageKit", "VersionMajor" ]).then(
-                                       function () {
-                                           client.features.packagekit = true;
-                                           return cockpit.resolve();
-                                       },
-                                       function () {
-                                           return cockpit.resolve();
-                                       });
+                    return cockpit.resolve();
                 });
```

Comment:
The function wrapper is not needed any more here, is it? I. e. just `.then(..., cockpit.resolve);` ?